src/Hydrodynamics/SphSimulation.cpp

```diff
@@ -651,18 +545,24 @@ void SphSimulation<ndim>::MainLoop(void)
   nbody->AdvanceParticles(n, nbody->Nnbody, t, timestep, nbody->nbodydata);
 
   // Check all boundary conditions
-  // (DAVID : Move this function to sphint and create an analagous one
-  //  for N-body.  Also, only check this on tree-build steps)
-  if (Nsteps%ntreebuildstep == 0 || rebuild_tree) sphint->CheckBoundaries(simbox,sph);
+  // (DAVID : create an analagous of this function for N-body)
```

Comment:
Yes, I will implement this in the icsilcc branch alongside the N-body Ewald commits.  Luckily the N-body Jeans test was only run for a few steps with no/little boundary crossings.